### PR TITLE
Sub Departments added

### PR DESCRIPTION
**Pull request by zafertaskent**

### CODE REVIEW

**Comment by zafertaskent** [comment]
@adilatilgan CR please, after this Department and SubDepartment separated.

File: api/policies/authenticated.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = function(req, res, next) {
   // User is allowed, proceed to controller
   var is_auth = req.isAuthenticated();
   console.log(req.path + ':' + req.method + '-' + is_auth);
-
+  return next();
```
Comment (adilatilgan) [inline api/policies/authenticated.js]: Why did you by pass authorization?

**Comment by adilatilgan** [comment]
I cannot see SubDepartment files here? May be you did not "git add" ?

File: api/policies/authenticated.js (hunk 1 above)
Comment (zafertaskent) [inline api/policies/authenticated.js]: Sorry I missed that while testing

**Comment by zafertaskent** [comment]
Sorry yes you are right should do  "git add", Fixin all in a minute.

**Comment by zafertaskent** [comment]
Can you check it now?

**Comment by adilatilgan** [comment]
LGTM now, make comments after your fixes please.

**Pull request merged**